### DISCUSSION TITLE
Is GitHub Education Pack is still available for Russian students?

### DISCUSSION THREAD

**Discussion by paks176** [community/community#61057] [GitHub Education] [answered]
### Hi everyone,

can't find information about is GitHub Education Pack is still available for Russian students or not. I found out JetBrains had  stopped support these students and there was no any announcements about it. What is the position of GitHub about this?
Sorry for bad English

**Comment by nirav-gajera** [comment, marked as answer]
No, the GitHub Education Pack is no longer available for Russian students. GitHub announced on February 25, 2022 that they would be suspending the GitHub Education Pack for students in Russia and Belarus due to the ongoing war in Ukraine. The suspension is effective immediately and will remain in place until further notice.

GitHub has said that they are committed to supporting all students, regardless of their location. However, they believe that it is important to take action in response to the war in Ukraine. The suspension of the GitHub Education Pack is one way that GitHub is showing their support for Ukraine and the Ukrainian people.

If you are a Russian student who is interested in learning more about coding, there are still other resources available to you. You can check out the following websites for more information:

* Free Code Camp: https://www.freecodecamp.org/
* The Odin Project: https://www.theodinproject.com/
* Codecademy: https://www.codecademy.com/

These websites offer free coding courses that you can take online. They also have communities of other students who are learning to code, so you can get help and support from others.

I hope this helps!

**Comment by buvanenko** [comment]
Of course, it was Russian students who started the war 🤡
What a hypocrisy.

![image](https://github.com/community/community/assets/60255961/0d6cc6a6-43ae-46e6-85c0-3d2926c48781)

**Reply by AbdrrahimDahmani** [reply]
That's unfair, such a shame to see such a behavior like this from an open source platform.

**Comment by zubairkhan111268** [comment]
Very good

**Comment by moonspeaking** [comment]
Видение GitHub — стать глобальной платформой для совместной работы разработчиков, независимо от того, где они находятся. 
_Мы серьезно относимся к нашей обязанности тщательно изучить правительственные санкции, чтобы убедиться, что пользователи и клиенты не пострадали сверх того, что требуется по закону_
Это также означает, что GitHub будет выступать за разработчиков в санкционированных регионах, чтобы они получили более широкий доступ к платформе и полный доступ к глобальному сообществу открытого исходного кода.

**Comment by danchistyakov** [comment]
Can we see any kind of official answer from Github about this?
